README.md

```diff
@@ -1,219 +1,251 @@
-    - `$ gcloud projects add-iam-policy-binding openmrs-260803 --role roles/healthcare.datasetAdmin --member serviceAccount:openmrs-fhir-analytics@openmrs-260803.iam.gserviceaccount.com`
-  * Activate the service account for your project using `gcloud auth activate-service-account <your-service-account> --key-file=<your-key-file> --project=<your project>`
-  * Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable: https://cloud.google.com/docs/authentication/getting-started#setting_the_environment_variable
- 
-7. Use the script [`utils/create_fhir_store.sh`](utils/create_fhir_store.sh) to create
-a FHIR store in this dataset which stream the changes to the BigQuery dataset
-as well:
-  `./utils/create_fhir_store.sh PROJECT LOCATION DATASET FHIR-STORE-NAME`
-  - `PROJECT` is your GCP project.
-  - `LOCATION` is GCP location where your dataset resides, e.g., `us-central1`.
-  - `DATASET` is the name of the dataset you created.
-  - `FHIR-STORE-NAME` is what it says.
-
-  *Note: If you get `PERMISSION_DENIED` errors, make sure to `IAM & ADMIN`/`IAM`/`Members` and add the `bigquery.dataEditor` and `bigquery.jobUser` roles to the `Cloud Healthcare Service Agent` service account that shows up.*
+```
+$ java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
+    org.openmrs.analytics.FhirEtl \
+    --openmrsServerUrl=http://localhost:8099/openmrs \
+    --openmrsUserName=admin --openmrsPassword=Admin123 \
+    --fhirSinkPath=http://localhost:8098/fhir \
+    --sinkUserName=hapi --sinkPassword=hapi \
+    --fileParquetPath=tmp/TEST/ \
+    --searchList=Patient,Encounter,Observation --batchSize=20
+```
 
-You can run the script with no arguments to see a sample usage. After you create
-the FHIR store, its full URL would be:
+Parameters:
 
-`https://healthcare.googleapis.com/v1/projects/PROJECT/locations/LOCATION/datasets/DATASET/fhirStores/FHIR-STORE-NAME`
-                    
-## Compile and run the streaming app
-From the root of your git repo, run:
+-   `searchList` - A comma-separated list of
+    [FHIR Search](https://www.hl7.org/fhir/search.html) URLs. For example,
+    `Patient?given=Susan` will extract only Patient resources that meet the
+    `given=Susan` criteria.
+-   `batchSize` - The number of resources to fetch in each API call.
+    Default: 100.
 
-`mvn clean -B install -DskipTests -pl streaming-atomfeed -am`
+### Batch mode using JDBC
 
-and then:
+To start Batch Mode using JDBC, run:
 
 ```
-mvn exec:java -pl streaming-atomfeed \
-    -Dexec.args=" --openmrsUserName=admin --openmrsPassword=Admin123 \
+$ java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
+    org.openmrs.analytics.FhirEtl \
     --openmrsServerUrl=http://localhost:8099/openmrs \
-    --fhirSinkPath=projects/PROJECT/locations/LOCATION/datasets/DATASET/fhirStores/FHIRSTORENAME \
-    --sinkUserName=hapi --sinkPassword=hapi "`
+    --openmrsUserName=admin --openmrsPassword=Admin123 \
```

Comment:
Suggestion: From this second command forward, drop the arguments with the default value and add a note upfront that we have not repeated the arguments with default values.

Reply:
I'm a little torn - I understand it's very repetitive, but the default values are not actually useful for end users, so listing them is an explicit reminder to swap in their own values.